pom.xml

```diff
@@ -871,6 +871,11 @@
                 <version>${guava.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
can we add it, before the test related dependencies?

Reply:
Sure will do.